<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenSidescan - by CIDCO</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Images/resources/cidco_icon.png</normaloff>:/Images/resources/cidco_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuObject_Inventory">
    <property name="title">
     <string>Ob&amp;ject Inventory</string>
    </property>
    <addaction name="actionFindObjects"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>&amp;Windows</string>
    </property>
    <addaction name="actionShowProjectWindow"/>
    <addaction name="actionShowFilePropertiesWindow"/>
    <addaction name="actionShowObjectInventoryWindow"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuObject_Inventory"/>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionFindObjects"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/resources/open.png</normaloff>:/Images/resources/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Sidescan File</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/resources/help.png</normaloff>:/Images/resources/help.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About OpenSidescan</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionFindObjects">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/resources/marker.png</normaloff>:/Images/resources/marker.png</iconset>
   </property>
   <property name="text">
    <string>Find &amp;Objects</string>
   </property>
  </action>
  <action name="actionShowProjectWindow">
   <property name="text">
    <string>Project</string>
   </property>
  </action>
  <action name="actionShowFilePropertiesWindow">
   <property name="text">
    <string>File Properties</string>
   </property>
  </action>
  <action name="actionShowObjectInventoryWindow">
   <property name="text">
    <string>Object Inventory</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionQuit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFindObjects</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionFindObjects()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>actionOpen()</slot>
  <slot>actionQuit()</slot>
  <slot>actionAbout()</slot>
  <slot>actionFindObjects()</slot>
  <slot>slot1()</slot>
 </slots>
</ui>
